sympy/printing/printer.py

```diff
@@ -360,9 +361,6 @@ def __init__(self, f, print_cls: Type[Printer]):
         self.__print_cls = print_cls
         update_wrapper(self, f)
 
-    def __repr__(self) -> str:
-        return repr(self.__wrapped__)  # type:ignore
```

Comment:
I think it would be good if `repr(latex)` contained either the word `latex`, `LatexPrinter`, or both - from that perspective, removing the repr entirely makes things worse.

Reply:
It does in Python 3.8:

```py
>>> latex
<sympy.printing.printer.latex_function object at 0x7fe1f9ac0110>
```

We can dynamically create the class in Python 3.9 as well if this seems important.

Reply:
For the sake of hiding the <3.9 hack, I'd be tempted to go with
```suggestion
    def __repr__(self):
        return f"_PrintFunction({self.__print_cls!r}, {self.__wrapped__!r})"

```
and the change I suggest below...